app/views/admin/items/new.html.erb

```diff
@@ -9,7 +9,7 @@
 
   <%= f.label :price%>
   <%= f.text_field :price%>
-
+# I've tried this a few times with different f. combos. Either nothing renders or there are a variety of nil class/merge/no conversion to key errors
```

Comment:
Line 16 looks pretty close to me. But, you may want to actually use checkboxes. Can't items be in multiple categories?

```
  <%= f.collection_check_boxes :category_ids, @categories, :id, :name do |cb| %>
      <% cb.label {cb.check_box + cb.text} %>
    <% end %>
```

Check out this stackoverflow for more on collection select: http://stackoverflow.com/questions/8907867/can-someone-explain-collection-select-to-me-in-clear-simple-terms